### ISSUE TITLE
After installation of B2S 2.0, plugins are not recognized

### ISSUE THREAD

**Issue by 7heS4ge** [vpinball/b2s-backglass#46]
I installed the new x86 version 2.0 in a new folder path (VisualPinball\B2SBackglassServer\).
I registered the new DLL and create folders Plugins and Plugins64. They contain DirectOuput shortcut and PinUpPlayer.
I checked "Activate plugins", but the button "Plugins Settings" cannot be clicked.
I change in the registry the value of Software\B2S\Plugins, but no effects.

I tried to debug the problem with Visual Studio on the project b2sbackglassserver, but how can I debug the DLL ? What should i run?

**Comment by JockeJarre** [comment]
to debug a dll you can specify the calling app.

https://learn.microsoft.com/en-us/visualstudio/debugger/how-to-debug-from-a-dll-project

Are you on VPU/discord? then please connect there @ Jarr3

**Comment by 7heS4ge** [comment]
Thank you for your reply.
No I'm not in Discord.
The calling app will be Visual Pinball ?
I must search a little more : I tried to launch the project b2sbackglassserverexe with a .directb2s file as parameter. I understood that when I click on the button "Plugin Settings" the window is not open by b2sbackglassserverexe, just registry keys are modified.

**Comment by JockeJarre** [comment]
Yes VPX calls it through the VB Scripts in the tables. The b2sbackglassserverregisterapp also instantiates it once, to find out if it was successful,
We could add a /test flag to b2sbackglassserverregisterapp, to make it do some small initial tests...

**Comment by JockeJarre** [comment]
Are you VPU maybe?

**Comment by 7heS4ge** [comment]
Thank you. I will try b2sbackglassserverregisterapp.
VPU for VPUniverse ? Yes : https://vpuniverse.com/profile/26257-jejevpuniverse/

**Comment by 7heS4ge** [comment]
I found the problem !
When I install B2S x86 on a folder named "B2SBackglassServer", plugins are not recognized (button Plugins Settings always grey) and the .directb2s file does not appear on the screen.
My previous path (don't work) : c:\Jeux\Visual Pinball\B2SBackglassServer\
But if I use another names, it's ok : c:\Jeux\Visual Pinball\B2SServer\

**Comment by JockeJarre** [comment]
this still sounds like magic to me... There is nothing in there (what I know) which should make this happen...
Apart from windows not wanting to long paths???

**Comment by JockeJarre** [comment]
So can you please close this issue?
